sass/app.scss

```diff
@@ -4,3 +4,57 @@
 $fa-font-path: "//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts";
 @import "font-awesome";
 @import "box-sizing";
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  width: 100%;
+  height: 50vh;
+  background: url(img/bg-login-top.png);
+  background-repeat: repeat;
+  align-content: center;
+}
+.logo {
```

Comment:
Since the logo is position absolute, you are having issues at smaller screen height sizes.

I would say try to make this work with just margin and padding.